.github/workflows/ios-build.yml

```diff
@@ -12,10 +12,10 @@ permissions:
 
 jobs:
   build-ios:
-    runs-on: macos-15-xlarge
+    runs-on: macos-15
```

Comment:
I had bumped us up to xlarge as an experiment a while ago, didn't mean to leave it on, it charges at like 2x the rate. I think with our low usage we were still within our free tier but best to stick to the 100% free size if we can!